bootstrap/fabric/src/main/java/org/geysermc/geyser/platform/fabric/GeyserFabricDumpInfo.java

```diff
@@ -43,21 +43,27 @@
 @Getter
 public class GeyserFabricDumpInfo extends BootstrapDumpInfo {
 
-    private String platformVersion = null;
```

Comment:
This should be kept, really it should probably be moved to `BootstrapDumpInfo`

Reply:
It wasnt removed, i just renamed it to loaderVersion since that better represents what it is - can revert

Reply:
yeah, i would just leave it as `platformVersion` to be in line with other platforms

Reply:
Renamed it back to platformVersion - i can't move it to BoostrapDumpInfo though since e.g. Standalone doesnt have that property.